lib/spack/docs/getting_started.rst

```diff
@@ -1514,3 +1514,203 @@ To ensure that Spack does not autodetect the Cray programming
 environment, unset the environment variable ``MODULEPATH``. This
 will cause Spack to treat a linux container on a Cray system as a base
 linux distro.
+
+.. _windows_support:
+
+----------------
+Spack On Windows
+----------------
+
+Windows support for Spack is currently under development.  While this work is still in an early stage,
+it is currently possible to set up Spack and perform a few operations on Windows.  This section will guide
+you through the steps needed to install Spack and start running it on a fresh Windows machine. 
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Step 1: Install prerequisites
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To use Spack on Windows, you will need the following packages:
+
+* Microsoft Visual Studio
+* Intel Fortran (needed for some packages)
+* Python 
+* Git
+
+"""""""""""""""""""""""
+Microsoft Visual Studio
+"""""""""""""""""""""""
+
+Microsoft Visual Studio provides the Windows C/C++ compiler that is currently supported by Spack.
```

Comment:
(question) is this the only supported compiler at the moment? If so, IMO it would be worth mentioning that here. I assume not though since there is mention of Intel oneAPI below and that comes with a C/C++ compiler (however, in that case, I'd assume this section would start out by saying you either need MSVC or Intel oneAPI).

Reply:
This is addressed by the second review PR.